power_grid_model_c/power_grid_model/include/power_grid_model/optimizer/tap_position_optimizer.hpp

```diff
@@ -69,11 +69,10 @@ constexpr void add_edges(main_core::MainModelState<ComponentContainer> const& st
             auto const& to_node = transformer3w.node(to_side);
             const TrafoGraphEdge edge_prop{main_core::get_component_idx_by_id(state, transformer3w.id()), 1};
 
-            const bool single_direction_condition =
-                regulated_objects.transformers3w.contains(transformer3w.id()) &&
-                (transformer3w.tap_side() == from_side || transformer3w.tap_side() == to_side);
+            auto const tap_at_from_side = transformer3w.tap_side() == from_side;
+            auto const single_direction_condition = regulated_objects.transformers3w.contains(transformer3w.id()) &&
+                                                    (tap_at_from_side || transformer3w.tap_side() == to_side);
```

Comment:
three winding transformer has `side_1`, `side_2` and `side_3`, which are also overloaded in the `ControlSide`. what's the reason for using `from` and `to` here for `3w`?